ngi_reports/common/ign_sample_report.py

```diff
@@ -62,11 +60,6 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
         self.parse_deliveries()
 
-        self.create_aggregate_statistics()
-
-        self.LOG.debug('Plotting graphs')
-        self.make_plots()
```

Comment:
I wanted to reuse the information scraping code in the `common.ign_sample_report.__init__` method for the aggregate report, but without generating the plots. As far as I could tell, there isn't anything in this package at least that uses this class directly. Do you think there may be something depending on this?